jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/util/Util.java

```diff
@@ -202,4 +209,74 @@ public static List<String> retrieveErrorRecords(String responseStr){
 		
 		return errorList;
 	}
+	
+	  /**
+     * This is inspired by java hash function.
+     *
+     * @param inStr the in str
+     * @return the unique id for string
+     */
+    public static String getUniqueIdForString(String inStr) {

```

Comment:
@arunpaladin this is copy-paste from here com.tmobile.pacman.commons.utils.CommonUtils
Why not to import that common utils instead of copy-pasting the code and create code duplication?

Reply:
removed above function. it is not used any more